### PR TITLE
[JS] feat: added bot framework debugger config

### PR DESCRIPTION
**Pull request by xtophs**
## Linked issues

#minor

## Details

added configuration to only start the bot to use with the bot framework emulator to launch.json in Teams Chef `js/samples /04.ai.a.teamsChefBot/.vscode/launch.json`

#### Change details

New launch config that does not start a brower and tried to sideload to teams.

**code snippets**:

        {
            "name": "Debug Local",
            "configurations": [
                "Attach to Local Service"
            ],
            "preLaunchTask": "Start application",
            "presentation": {
                "group": "all",
                "order": 2
            },
            "stopAll": true
        }

## Attestation Checklist

- [X] My code follows the style guidelines of this project

- I have checked for/fixed spelling, linting, and other errors
- I have commented my code for clarity
- I have made corresponding changes to the documentation (we use [TypeDoc](https://typedoc.org/) to document our code)
- My changes generate no new warnings
- I have added tests that validates my changes, and provides sufficient test coverage. I have tested with:
  - Local testing
  - E2E testing in Teams
- New and existing unit tests pass locally with my changes

### Additional information

> Feel free to add other relevant information below

### CODE REVIEW

**Review by corinagum — APPROVED** [review]

File: js/samples/04.ai.a.teamsChefBot/.vscode/launch.json [hunk 1]
```diff
@@ -90,6 +90,18 @@
                 "order": 2
             },
             "stopAll": true
+        },
+        {
+            "name": "Debug Local",
```
Comment (corinagum) [inline js/samples/04.ai.a.teamsChefBot/.vscode/launch.json:95]: I think we want to indicate that this is for the emulator
```suggestion
            "name": "Debug Local in BF Emulator",
```

**Review by corinagum — CHANGES_REQUESTED** [review]
thank you @xtophs! Do you think it would be beneficial to have this for all of the samples? I'm also wondering if we want to add a screencap to the sample startup documentation with this information. What are your thoughts?

**Comment by singhk97** [comment]
I like the idea of testing the samples without sideloading it into Teams. Teams Toolkit has implemented the [`Teams Test Tool`](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/debug-your-teams-app-test-tool) that does just that. You can try it out from the "AI Chat bot" sample from the Teams Toolkit VS code sample: 
image

The BotFramework Emulator is great but you would have to install the application to use it, while the test tool is an npm library and no additional set up is needed from the user.

@corinagum What are your thoughts on this? I think we could add it to one sample first and then slowly roll out to the other ones.

P.S. The test tool is also available in Visual Studio 17.9 Preview 3 onwards.

**Comment by xtophs** [comment]
I found the inner loop workflow to be faster without side-loading into teams. Once the bot framework is installed and the emulator is launched and pointed to the localhost endpoint, you can make code changes and test instantaneously. 

Launching the browser and sideloading updates isn't quite as smooth. 

Yes, it would make sense to add this config to all the samples.

**Comment by corinagum** [comment]
@xtophs I'm going to close this PR for now - please feel free to open a new one once the 2 PRs you made are combined. We'd also like to differentiate between running locally BF Emulator.

**Pull request closed without merging**